NineChronicles.Headless/GraphTypes/RuneSlotInfoInputType.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using GraphQL.Types;
+using Libplanet;
+using Libplanet.Assets;
+using Nekoyume.Action;
+
+namespace NineChronicles.Headless.GraphTypes
+{
+    public class RuneSlotInfoInputType : InputObjectGraphType<RuneSlotInfoType>
```

Comment:
Maybe it should be `RuneSlotInfo`, not GraphQL type, isn't it?

```suggestion
    public class RuneSlotInfoInputType : InputObjectGraphType<RuneSlotInfo>
```